docs/ios/v2/features/change-role.mdx

```diff
@@ -43,3 +43,37 @@ hmsSDK.accept(changeRole: roleChangeRequest)
 which completes the `changeRole` loop. Both parties will receive a `roleUpdated` callback so that they both can do necessary UI updates. Now the user actually becomes a speaker and the audio publishing will start automatically.
 
 Now lets imagine the newly nominated speaker is not behaving nicely and we want to move him back to listener without a prompt. This is where the `force` parameter comes in. When it is set to `true` the other party will not receive a confirmation `roleChangeRequest` but instead will straight away receive a new set of updated permissions and stop publishing. `roleUpdated` callback will still be fired so that the app can update the user's UI state.
+
+## Bulk Role Change
+Bulk role change is used when you want to convert all roles from a list of roles, to another role.

```

Comment:
* when you want to change role of all users from a list of roles, to another role.